src/js/order.js

```diff
@@ -646,7 +645,7 @@ function getOrdersForYearAndQueryTemplate(
 }
 
 function fetchYear(year, request_scheduler, nocache_top_level) {
-    const templates_by_site = {
```

Comment:
I can live with this but offer an alternative that reduces the chance of misbehaviour later:
keep it const here and include the "other" element. The object isn't actually immutable, but by convention the const reference signals such.
Then probe for the country and if it's not there, use other.
If you hate it, I'll just merge your version.
P

Reply:
What about digital orders?

Reply:
It looks like either I don't know how to do reviews in github or you've not pushed changes - I see you've closed the spelling mistake  comment but the mistake is still there. Am I being dense?
On this one, it looks like you've not changed anything. I don't understand the bit about digital orders = your "other" template url param has digital=1 - am I missing something?

Reply:
Are .com and .mx different from all others? For .com, physical orders and digital orders are separate lists. I noticed the issue when I commented out www.amazon.com and smile.amazon.com, and tested (inadvertently) with smile.amazon.com; digital orders didn't get reported. When I reinstated and tested again, still no digital for smile.amazon.com; I had to add the second URL template under smile.amazon.com to get the digital orders to appear.

Reply:
I locked down the language for all of the sites with the last commit. For example, .ca offers French, but the extension only supports English, so a French user would get a failure; this change should prevent that.

Reply:
Some testing... these are the only combinations which give digital-only orders for .com:
&unifiedOrders=0
&digitalOrders=1&unifiedOrders=0
&digitalOrders=1&unifiedOrders=

All other combinations give only physical orders.

Do other sites work differently?

Reply:
yep - all of the patterns in the template object have worked at some point. You'll notice that the US ones are the strange ones - I guess this is probably because the US site was the first.

Reply:
I don't have an mx account so I can't test it - I think it came from a contributor. Getting test dumps from users is key (in my view) to being able to sort country related issues out.